### ISSUE TITLE
Update Save to RXN for pathway reactions

### ISSUE THREAD

**Issue by ivanmaslow** [epam/Indigo#2234]
The first step(if it will be simpler from a development perspective): 
- If a user tries to save a pathway reaction to RXN file format - user can't, we should display an error that the user can't save any multi-tail arrow to RXN (like saving arrow to SDF)
- Saving/loading of pathway reaction to other formats (Smart, Smiles, CML) should work as for .RXN

The second step (separate task):
- User can save one multi-tail (simple reaction) to RXN, after loading from RXN it will be displayed as a simple reaction (plus and arrow) 
- When user saves several reactions or several multi-tail arrows, a warning should be displayed that we can't save multi-tails or multi reactions to RXN, only **the first root reaction is saved** (or we can't save - need to discuss after investigation of ChemDraw behavior for several reactions in RXN - https://github.com/epam/Indigo/issues/2194)

**Comment by ivanmaslow** [comment]
Discussed with @AlexanderSavelyev - it's blocked by https://github.com/epam/Indigo/issues/2235

**Comment by yauhen-karmyzau** [comment]
Implemented in scope #2237

**Comment by ivanmaslow** [comment]
Exploratory testing is completed on RC Remote environment:
- Ketcher [Version 2.26.0-rc.1] Build at 2024-10-04; 18:45:04
- Indigo Version 1.25.0-rc.1.0-g704673fcb-x86_64-linux-gnu-11.2.1
- Desktop, Windows 11 Enterprise
- Chrome Version 129.0.6668.100 (Official Build) (64-bit)

Points below are covered during testing, tests are updated:
- If user added Multi-Tailed Arrow to Canvas, option of saving to RXN V2000/V3000 is available:
![image](https://github.com/user-attachments/assets/130c1010-a5fa-43b0-9218-32e44aab1582)
- If different types of reactions (single, cascade, combinations) are on Canvas, after saving to RXN V2000/V3000 the only one root reaction is saved, structure of the file is displayed on preview:
![image](https://github.com/user-attachments/assets/50ea6e3b-cf2d-4e57-8321-35be1e41a41b)
- Single reactions with Multi-Tailed arrow (1) can be saved to MDL Rxnfile V2000/MDL Rxnfile V3000/Daylight Smarts/Daylight SMILES/Extended SMILES, but after loading they will be displayed (2) with single arrows and pluses:
![image](https://github.com/user-attachments/assets/93b2d6cf-14b0-4b4e-885f-5071d10c093f)
- Reactions with Multi-Tailed or single arrows can be loaded from ket, RDF or added by tools and after that saved to RXN V2000/V3000

During exploratory testing a huge part of tests fail because of opened defects, they will be fixed according to priority and selected milestones. Retesting is required after fixing because the core functionality is affected:
- https://github.com/epam/Indigo/issues/2462
- https://github.com/epam/Indigo/issues/2485
- https://github.com/epam/Indigo/issues/2482
- https://github.com/epam/Indigo/issues/2493
